src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -106,11 +105,6 @@ public EditorPanel getEditor() {
   }
 
   @Override
-  public boolean sameType(final IBean other) {
-    return other instanceof PropertiesDiceRoller && getDisplayName().equals(other.getDisplayName());
```

Comment:
This comparison is slightly different than all the other implementations that simply compare the classes of the two beans.  Anything to be concerned about?  Can multiple instances of `PropertiesDiceRoller` exist concurrently?  What if we have multiple `.properties` files under `/dice_servers`?

Reply:
Right...
I changed the equals and hash code implementation to fix this issue...